OpenDBDiff/Front/PrincipalForm.Designer.cs

```diff
@@ -208,28 +206,28 @@ private void InitializeComponent()
             this.txtOldObject.Styles.LineNumber.Size = 9F;
             this.txtOldObject.Styles.Max.Size = 9F;
             this.txtOldObject.TabIndex = 0;
-            //
+            // 

```

Comment:
Please undo all the whitespace changes.

Reply:
Is this something covered by .editorconfig or CodeMaid? If this work was done before those, I vote to just let it ride in the .designer.cs files -- do a one-time whitespace cleanup pull request and then lean on .editorconfig/CodeMaid moving forward.